cobalt/watchdog/watchdog.h

```diff
@@ -89,7 +89,8 @@ class Watchdog : public Singleton<Watchdog> {
   bool Unregister(const std::string& name, bool lock = true);
   bool Ping(const std::string& name);
   bool Ping(const std::string& name, const std::string& info);
-  std::string GetWatchdogViolations(bool clear = true);
+  std::string GetWatchdogViolations(const char* clients[] = {},
```

Comment:
Can we make this string to keep it consistent with other functions in the class?